packages/health-check-runner/index.ts

```diff
@@ -0,0 +1,61 @@
+const exec = require("child_process").exec;
+import { Logger } from "@uma/financial-templates-lib";
+
+let logger;
+
+export async function run(): Promise<void> {
+  logger = Logger;
+
+  logger.debug({ at: "HealthCheckRunner", message: "Starting health check runner 🩺" });
+
+  const healthCheckCommands = process.env.HEALTH_CHECK_COMMANDS ? JSON.parse(process.env.HEALTH_CHECK_COMMANDS) : [];
+
+  if (healthCheckCommands.length === 0) {
+    logger.debug({ at: "HealthCheckRunner", message: "No health check commands to run. Closing" });
+    return;
+  }
+
+  logger.debug({ at: "HealthCheckRunner", message: "Running health check commands", healthCheckCommands });
+
+  const outputs = await Promise.all(healthCheckCommands.map(async (command) => execShellCommand(command, process.env)));
+  const errorOutputs = outputs.filter((output) => output.error);
+  const validOutputs = outputs.filter((output) => !output.error);
+  const outputLogLevel = errorOutputs.length > 0 ? "error" : "debug";
+
+  logger[outputLogLevel]({
+    at: "HealthCheckRunner",
+    message: `Health check command ${outputLogLevel == "error" ? "failed" : "succeeded"}!`,
+    validOutputs: validOutputs.map((output) => output.cmd),
+    errorOutputs,
```

Comment:
Have we tested to ensure this list of errors stringifies correctly with the logger? Should we pre-stringify to avoid the risk of `[Object object]` type outputs?

Reply:
I've tested it running hardhat scripts that error if there is a problem and foundry tests against a mainnet fork.

Reply:
Sounds good, as long as you're confident the error list is _always_ stringified in a readable way, this SGTM!

This is probably the most important output of this, so it's super important that the person being notified can read the errors and all of their properties no matter the environment where it's run.